samples/matter/lock/CMakeLists.txt

```diff
@@ -17,7 +17,11 @@ if(CONF_FILE)
 endif()
 
 if(NOT CONF_FILE_NAME STREQUAL "prj_no_dfu.conf")
-    set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static_dfu.yml)
+    if(NOT CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE)
+        set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static_dfu.yml)
+    else()
+    set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static_dfu_wifi_ext_patch.yml)
+    endif()
```

Comment:
this will never work because zephyr isn't loaded at this poing so no project configuration has been loaded

Reply:
Oh, right.